authz/sdk_end2end_test.go

```diff
@@ -420,31 +420,29 @@ func (s) TestSDKFileWatcher_ValidPolicyRefresh(t *testing.T) {
 	if err := os.WriteFile(file, []byte(valid2.authzPolicy), os.ModePerm); err != nil {
 		t.Fatalf("os.WriteFile(%q) failed: %v", file, err)
 	}
-	// Wait 30 ms for background go routine to read the updated files.
-	time.Sleep(30 * time.Millisecond)
 
 	// Verifying authorization decision.
-	_, err = client.UnaryCall(ctx, &pb.SimpleRequest{})
 	numRetries := 0
 	reloadSuccess := false
```

Comment:
Optional: this variable is unnecessary.  Fail unconditionally after the loop and `return` in the loop instead of `break`ing.

Reply:
Good point. Acknowledged.